src/xmlnode.cpp

```diff
@@ -204,6 +209,12 @@ bool XmlNode::hasAttribute(const char *attributeName) const
     return found;
 }
 
+xmlNsPtr getAttributeNamespace(const xmlNodePtr &node, const char *attributeName)
+{
+    xmlAttrPtr attribute = xmlHasProp(node, reinterpret_cast<const xmlChar *>(attributeName));
+    return attribute->ns;
```

Comment:
Not a big deal (I guess), but I would simplify this with:

```
return xmlAttrPtr attribute = xmlHasProp(node, reinterpret_cast<const xmlChar *>(attributeName))->ns;
```

Reply:
Done